verification/curator-service/api/test/auth.test.ts

```diff
@@ -0,0 +1,17 @@
+import app from '../src/index';
+import request from 'supertest';
+
+describe('auth', () => {
+    it('login with google', (done) => {
+        // Redirected to the oauth consent screen.
+        request(app).get('/auth/google').expect(302, done);
+    });
+    it('logout', (done) => {
+        // Redirects to /
+        request(app).get('/auth/logout').expect(302, done);
```

Comment:
Is there a way to check it redirected to the correct URL?

Reply:
Yes I added check for the location where it was possible, I did not add it for the oauth consent screen as that's hosted by google and it could change and would break tests.